Lib/test/test_concurrent_futures/test_wait.py

```diff
@@ -42,8 +42,12 @@ def test_first_completed(self):
         self.assertEqual(set([future1]), done)
         self.assertEqual(set([CANCELLED_FUTURE, future2]), not_done)
 
+        event.set()
```

Comment:
Use maybe try/finally to unblock the task on test failure? Same remark for other tests.